pgaudit.c

```diff
@@ -984,6 +988,19 @@ log_select_dml(Oid auditOid, List *rangeTabls)
         Oid relOid;
         Relation rel;
         RangeTblEntry *rte = lfirst(lr);
+        
+        /* 
+         * [Hope] We only show the table in the query, 
+         * there is no need to show partitions or inheritance child. 
```

Comment:
No need to put your name in the comments.  The git log will show who modified what.

Reply:
yes, should remove the name.